enforcer-parent/commons/src/test/java/org/wso2/choreo/connect/enforcer/commons/model/RequestContextTest.java

```diff
@@ -72,8 +72,14 @@ public void testPathParametersWithNumbers() {
     public void testPathParameterGenerationWithWildcard() {
         testPathParamValues("/v2/pet/12/2/random/random2", "/v2", "/pet/{petId}/{imageId}/*",
                 "petId", "12");
+        testPathParamValues("/v2/pet/12/2/random/random2/random3", "/v2", "/pet/{petId}/{imageId}/*",
+                "petId", "12");
         testPathParamValues("/v2/pet/12/image/2/random/random2", "/v2", "/pet/{petId}/image/{imageId}/*",
                 "imageId", "2");
+        testPathParamValues("/v2/pet/12/2/image", "/v2", "/pet/{petId}/{imageId}/image/*",
+                "petId", "12");
+        testPathParamValues("/v2/pet/12/2/image123", "/v2", "/pet/{petId}/{imageId}/image/*",
+                null, null);
```

Comment:
Are we allowing the following? shall we add following cases as well if not added already?
```suggestion
        testPathParamValues("/v2/pet/12/2/image", "/v2", "/pet/{imageId}/image",
        "imageId", "12/2"); 
        testPathParamValues("/v2/pet/12/image/*", "/v2", "/pet/{imageId}/image/*",
        "imageId", "12"); 
```

Reply:
added those test cases.
except 
`testPathParamValues("/v2/pet/12/2/image", "/v2", "/pet/{imageId}/image","imageId", "12/2");` - invalid
`testPathParamValues("/v2/pet/12/image/*", "/v2", "/pet/{imageId}/image/*", "imageId", "12"); ` - already covered.